embedded/sql/stmt.go

```diff
@@ -1668,6 +1669,75 @@ func (v *Blob) Compare(val TypedValue) (int, error) {
 	return bytes.Compare(v.val, rval), nil
 }
 
+type Float struct {
+	val float64
+}
+
+func (v *Float) Type() SQLValueType {
+	return FloatType
+}
+
+func (v *Float) IsNull() bool {
+	return false
+}
+
+func (v *Float) inferType(cols map[string]ColDescriptor, params map[string]SQLValueType, implicitDB, implicitTable string) (SQLValueType, error) {
+	return FloatType, nil
+}
+
+func (v *Float) requiresType(t SQLValueType, cols map[string]ColDescriptor, params map[string]SQLValueType, implicitDB, implicitTable string) error {
+	if t != TimestampType {
```

Comment:
FloatType

Reply:
and probably Integer can be converted into Float, but that would be in integer method